bin/run_functions.js

```diff
@@ -496,6 +493,10 @@ function executeProject(filename, project, options) {
                         }
                     }
                     jobId = null;
+
```

Comment:
Relocating the fail/exit logic here ensures that:

1. Reports are still run even for failed runs and
2. jobId is set to null which avoids the "To cancel the job started, please use..." message